dev/com.ibm.ws.microprofile.rest.client.FT_fat/fat/src/com/ibm/ws/microprofile/rest/client/fat/FATSuite.java

```diff
@@ -17,49 +17,43 @@
 import org.junit.runners.Suite;
 import org.junit.runners.Suite.SuiteClasses;
 import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.JakartaEE9Action;
 
 @RunWith(Suite.class)
 @SuiteClasses({
                 RetryTest.class,
                 TimeoutTest.class
 })
 public class FATSuite {
-    private static final String[] ALL_VERSIONS = {"1.0", "1.1", "1.2", "1.3", "1.4", "2.0"};
+    private static final String[] ALL_VERSIONS = {"1.0", "1.1", "1.2", "1.3", "1.4", "2.0", "3.0"};
 
-    static FeatureReplacementAction MP_REST_CLIENT(String version, String serverName) {
-        return MP_REST_CLIENT(new FeatureReplacementAction(), version, serverName);
-    }
-
-    static FeatureReplacementAction MP_REST_CLIENT(FeatureReplacementAction action, String version, String serverName) {
-        return use(action, "mpRestClient", version)
-                        .withID("mpRestClient-" + version)
-                        .forServers(serverName);
-    }
-
-    static FeatureReplacementAction MP_REST_CLIENT_WITH_CONFIG(String version, String serverName) {
-        return MP_REST_CLIENT_WITH_CONFIG(new FeatureReplacementAction(), version, serverName);
-    }
-
-    static FeatureReplacementAction MP_REST_CLIENT_WITH_CONFIG(FeatureReplacementAction action, String version, String serverName) {
-        action = use(action, "mpRestClient", version)
-                        .withID("mpRestClient-" + version)
-                        .forServers(serverName);
+    private static FeatureReplacementAction MP_REST_CLIENT_WITH_CONFIG(FeatureReplacementAction action, String version, String serverName) {
+        action = use(action, "mpRestClient", version).forServers(serverName);
         switch(version) {
             case "1.0":
-            case "1.1": return use(action, "mpConfig", "1.1", "1.0", "1.2", "1.3", "1.4", "2.0");
+            case "1.1": return use(action, "mpConfig", "1.1", "1.0", "1.2", "1.3", "1.4", "2.0", "3.0");
             case "1.2":
-            case "1.3": return use(action, "mpConfig", "1.3", "1.0", "1.1", "1.2", "1.4", "2.0");
-            case "1.4": return use(action, "mpConfig", "1.4", "1.0", "1.1", "1.2", "1.3", "2.0");
-            case "2.0":
-            default:    return use(action, "mpConfig", "2.0", "1.0", "1.1", "1.2", "1.3", "1.4");
+            case "1.3": return use(action, "mpConfig", "1.3", "1.0", "1.1", "1.2", "1.4", "2.0", "3.0");
+            case "1.4": return use(action, "mpConfig", "1.4", "1.0", "1.1", "1.2", "1.3", "2.0", "3.0");
+            case "2.0": return use(action, "mpConfig", "2.0", "1.0", "1.1", "1.2", "1.3", "1.4", "3.0");
+            case "3.0":
+            default:    return use(action, "mpConfig", "3.0", "1.0", "1.1", "1.2", "1.3", "1.4", "2.0");
```

Comment:
This isn't wrong but I would have recommended using the MicroProfileActions class